src/clj/game/cards/ice.clj

```diff
@@ -1451,6 +1451,33 @@
                                                                    (shuffle! :deck))}
                                                  card nil)))}]}
 
+   "Mlinzi"
+   (letfn [(net-or-trash [net-dmg mill-cnt]
+             {:label (str "Do " net-dmg " net damage unless the Runner trashes the top "
```

Comment:
I think this should just read "Do X net damage". The "unless..." part is irrelevant to the Corp when they are identifying which subroutine to resolve, which is the point of the ability label.